NuGet.Config

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Please reset this file back to the dev branch copy, since there are no "real" changes in it.